tools/runners/common.sh

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/env bash
+
+# SDL files to back-up
+SDL_BACK_UP=("sdl_preloaded_pt.json" "smartDeviceLink.ini" "hmi_capabilities.json" "log4cxx.properties")
+
+# ATF and SDL files/folders to remove before each script run
+ATF_CLEAN_UP=("sdl.pid" "mobile*.out")
+SDL_CLEAN_UP=("*.log" "app_info.dat" "storage" "ivsu_cache" "../sdl_bin_bk")
+
+logf() { log "$@" | tee -a ${REPORT_PATH_TS}/${REPORT_FILE}; }
+
+remove_color() { sed -i "s/\x1b[^m]*m//g" $1; }
+
+status() {
+  logf ${LINE1}
+  logf "Test target:" $TEST_TARGET
+  logf ${LINE2}
+  for i in ${LIST_TOTAL[@]}; do logf "${i}"; done
+  logf ${LINE2}
+  logf "TOTAL: " $ID
+  logf "${P}PASSED: " ${#LIST_PASSED[@]} "${N}"
+  logf "${F}FAILED: " ${#LIST_FAILED[@]} "${N}"
+  logf "${A}ABORTED: " ${#LIST_ABORTED[@]} "${N}"
+  logf "${S}SKIPPED: " ${#LIST_SKIPPED[@]} "${N}"
+  logf ${LINE2}
+  logf "Execution time:" $(seconds2time $(($ts_finish - $ts_start)))
+  logf ${LINE1}
+  log
+  remove_color ${REPORT_PATH_TS}/${REPORT_FILE}
+}
+
+process() {
+  ID=0
+  local EXT=${TEST_TARGET: -3}
+  if [ $EXT = "txt" ]; then
+    local total_num_of_scripts=$(wc -l $TEST_TARGET | awk '{print $1}')
+    while read -r ROW; do
+      local script=$(echo $ROW | awk '{print $1}')
+      local issue=$(echo $ROW | awk '{print $2}')
+      run $script $total_num_of_scripts $issue
+    done < "$TEST_TARGET"
+  elif [ $EXT = "lua" ]; then
+    run $TEST_TARGET 1
+  else
+    local LIST=($(find $TEST_TARGET -iname "[0-9]*.lua" | sort))
+    for ROW in ${LIST[@]}; do
+      run $ROW ${#LIST[@]}
+    done
+  fi
+}
+
+run_atf() {
+  local SCRIPT=$1
+  local NUM_OF_SCRIPTS=$2
+  local ISSUE=$3
+
+  local ID_SFX=$(printf "%0${#NUM_OF_SCRIPTS}d" $ID)
+  if [ -n "$TEST_ID" ]; then ID_SFX=$TEST_ID; fi
+
+  REPORT_PATH_TS_SCRIPT=${REPORT_PATH_TS}/${ID_SFX}
+  mkdir ${REPORT_PATH_TS_SCRIPT}
+
+  local RESULT_CODE=4
```

Comment:
```suggestion
  local RESULT_CODE=3
```
This marks all comments as skipped tests, these lines should just be ignored

Reply:
@jacobkeeler Please notice `3` is already reserved:
https://github.com/smartdevicelink/sdl_atf/blob/971c3b80bc976fb8f87d8f122b9229be64140756/modules/exit_codes.lua#L20

Reply:
Also some improvement regarding comment lines has been made in b13bdc5